data/types/basal/temporary/temporary.go

```diff
@@ -182,10 +186,12 @@ func parseSuppressed(parser structure.ObjectParser) Suppressed {
 func validateSuppressed(validator structure.Validator, suppressed Suppressed) {
 	if suppressed != nil {
 		switch suppressed := suppressed.(type) {
+		case *dataTypesBasalAutomated.SuppressedAutomated:
```

Comment:
Both case statements can be merged since they use the same validator, e.g.:
```golang
switch suppressed := suppressed.(type) {
case *dataTypesBasalAutomated.SuppressedAutomated,
     *dataTypesBasalScheduled.SuppressedScheduled:
         suppressed.Validate(validator)
}
```

Reply:
True. I think I'd like to keep it as-is so the `validateSuppressed` function follows the same form as the `parseSuppressed` function just above it (which cannot combine case statements).